### DISCUSSION TITLE
When starting the desktop version, I want it to remember the position of the previous window

### DISCUSSION THREAD

**Discussion by kako774** [ente-io/ente#501] [Enhancements]
When starting the desktop version, the previous window position is not remembered, so I would like it to be remembered.

**Comment by j7e** [comment]
+1 for this on macOS.

When opening the application, I want it to use the same window position and size (and monitor/desktop where there are multiple) from when the application was last closed. The window should also appear on top of other applications' windows rather than underneath them (as sometimes happens when I open it).

Similar requests open with #5348 and #2181.
